config/routes.rb

```diff
@@ -50,6 +50,10 @@
 
   get "/documentation" => "documentation#index"
 
+  get "/publisher-information/how-to" => "publisher_information#how_to", as: :how_to
```

Comment:
It might be worth rethinking this route name it feels pretty generic as just "how_to" could be something like "usage_documentation" ?

Reply:
I've changed all references to how-to to how-to-use-publisher, which will hopefully make the code more readable